site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: blog-post
+title: How to get the most out of a trial period
+description: When you are trialing software, how can you make most effective use of this time?
```

Comment:
When you are trialing auth software, how can you make most effective use of this time?